muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/discussion/body/application/current-thread.tsx

```diff
@@ -139,17 +150,17 @@ class CurrentThread extends React.Component<CurrentThreadProps, CurrentThreadSta
 
         this.props.discussion.currentReplies.map((reply: DiscussionThreadReplyType) => {
           const user: DiscussionUserType = reply.creator;
-          const userCategory = reply.creator.id > 10 ? reply.creator.id % 10 + 1 : reply.creator.id;
-          const canRemoveMessage = this.props.userId === reply.creator.id || areaPermissions.removeThread;
-          const canEditMessage = this.props.userId === reply.creator.id || areaPermissions.editMessages;
+          const userCategory: number = reply.creator.id > 10 ? reply.creator.id % 10 + 1 : reply.creator.id;
```

Comment:
Same here about the pharentesis, while the ternary is clear on priority, the % and + are not that clearly know, makes it bit hard to read to humans (at least to me).